rosdep/python.yaml

```diff
@@ -19,7 +19,17 @@ adafruit-mcp3008-pip:
   ubuntu:
     pip:
       packages: [Adafruit-MCP3008]
-adafruit-pca9685-pip:
```

Comment:
While the advice that @sloretz gave you here was correct for new keys, we shouldn't change the old ones (even if they don't meet our current standards).  That's because it might break people who are already using this.  Please put this one back to:

```
adafruit-pca9685-pip
```